apps/scan/backend/scripts/copy_sheets.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import path from 'path';
+import { v4 as uuid } from 'uuid';
+import { AcceptedSheet } from '@votingworks/backend';
+import {
+  assert,
+  assertDefined,
+  extractErrorMessage,
+} from '@votingworks/basics';
+import { safeParseNumber } from '@votingworks/types';
+
+import { SCAN_WORKSPACE } from '../src/globals';
+import { Store } from '../src/store';
+
+const usageMessage = `Usage: copy-sheets <target-sheet-count>
+
+Arguments:
+  <target-sheet-count>\tThe target total sheet count after copying`;
+
+interface CopySheetsInput {
+  targetSheetCount: number;
+}
+
+function checkEnvironment(): void {
+  assert(process.env.NODE_ENV !== 'production', 'Cannot be run in production');
+}
+
+function parseCommandLineArgs(args: readonly string[]): CopySheetsInput {
+  if (args.length !== 1 || !safeParseNumber(args[0]).isOk()) {
+    console.log(usageMessage);
+    process.exit(0);
+  }
+  const targetSheetCount = safeParseNumber(args[0]).unsafeUnwrap();
+  return { targetSheetCount };
+}
+
+function copySheet(store: Store, sheet: AcceptedSheet): void {
+  const newSheetId = uuid();
+  const newSheet: AcceptedSheet = {
+    ...sheet,
+    id: newSheetId,
+    frontImagePath: sheet.frontImagePath.replace(sheet.id, newSheetId),
+    backImagePath: sheet.backImagePath.replace(sheet.id, newSheetId),
+  };
+
+  fs.copyFileSync(sheet.frontImagePath, newSheet.frontImagePath);
+  fs.copyFileSync(sheet.backImagePath, newSheet.backImagePath);
+
+  store.addSheet(newSheetId, newSheet.batchId, [
+    {
+      imagePath: newSheet.frontImagePath,
+      interpretation: newSheet.interpretation[0],
+    },
+    {
+      imagePath: newSheet.backImagePath,
+      interpretation: newSheet.interpretation[1],
+    },
+  ]);
+}
+
+function copySheets({ targetSheetCount }: CopySheetsInput): void {
+  const store = Store.fileStore(
+    path.join(assertDefined(SCAN_WORKSPACE), 'ballots.db')
+  );
+  const sheets = Array.from(store.forEachAcceptedSheet());
```

Comment:
This could load quite a lot of data. Consider getting the length first, determining the number of sheets to create, then loading only as many as might be needed but up to some limit, like 500.

Reply:
Good call: https://github.com/votingworks/vxsuite/pull/4225/commits/abb2dbdd663c5bad282cdef21c54d088f7d6737f